.github/workflows/default-dependabot-automerge-whitelist.conf

```diff
@@ -5,3 +5,5 @@ org.junit:junit-bom minor
 org.assertj:assertj-core minor
 com.adobe.testing:s3mock-testcontainers minor
 org.testcontainers:testcontainers-bom minor
+org.testcontainers:testcontainers-bom minor
+org.wiremock:wiremock-standalone minor
```

Comment:
why minor?

Reply:
[The releases are typically minor, rarely patch](https://mvnrepository.com/artifact/org.wiremock/wiremock-standalone).